server.js

```diff
@@ -26,9 +27,19 @@ const server = new ApolloServer({
   schema,
   context: async ({ req }) => {
     const { token } = req.headers || '';
+
+    logger.log({
+      level: 'info',
+      message: `method: ${req.method}/baseUrl: ${req.baseUrl}/date:${
+        req.fresh
+      }/ request headers: ${JSON.stringify(req.headers)}/body: ${JSON.stringify(
+        req.body
+      )}`,
+    });
     if (token) {
       const user = verifyUser(token);
       if (!user) throw new AuthenticationError('Invalid authorization token');
```

Comment:
Please, while you have time, adjust the error message to use `INVALID_AUTHORIZATION_TOKEN` in exception, tests if it is checked, and the logger message. I think it is a good idea to create a format function to remove the `_` and lowercase text from our standard error messages